### ISSUE TITLE
Want to return string instead of key

### ISSUE THREAD

**Issue by fingertips-solutions** [tohidplus/laravel-vue-translation#4]
Hi,

If the key is present only in one language say ar and same key is not in en file then on en version its showing key like message.foo.bar i want it if key is not there on english version then display actual string. can you help me with that ?

**Comment by tohidplus** [comment]
Can you please explain a little more?
I don't understand, If the key doesn't exist then which string should be displayed??

**Comment by danish-saee** [comment]
Thank you for getting back to me.  here are some more details. 

Basically when i have strings in ar/messages.php like below

return [
    "Total Captain" => "Total Captain Arabic",
    "Dashboard" => "Dashboard Arabic"
];

when i try to get messages.Total Captain its getting translated in ar language. but if i do not have this key "Total Captain" on en/messages.php then when i go to en version its showing me messages.Total Captain that is key it self. 

what i want is if this messages.Total Captain is not there it should show Total Captain in this way i have to manage 2 files one in ar/messages.php and 2nd in en/messages.php

Basically i want to manage single file i am not sure if its possible or not ?

**Comment by tohidplus** [comment]
You can edit the `Translation.js` file like this:

```javascript

const translations = require('./translations');
export default {
    translate(key,replacements={}) {
        let lang = document.documentElement.lang;
        let word = translations[lang];
        const keys = key.split('.');
        for (let i in keys) {
            try {
                word = word[keys[i]];
                if (word === undefined) {
                    // One change
                    word = keys[i];
                    break;
                }
            } catch (e) {
                // Second change
                word = keys[i];
                break;
            }
        }
         for (let i in replacements){
             word=word.replace(`:${i}`,replacements[i]);
         }
         return word;
    }
}

```
